src/test/resources/features/group.feature

```diff
@@ -41,3 +79,20 @@ Feature: Endpoint group
       | isBroadcast           | false              |
       | isBroadcast           | true               |
       | canHaveChatterGuests  | false              |
+
+  @PatchGroup
+  Scenario Outline: PATCH to group fail
+    Given I build "PATCH" request to group
+    When I use endpoint "chatter/groups/{groupId}" request to group with "groupId"
+    And I update group the "<parameterToUpdate>" to "<updateDate>"
+    Then the response status code should be "BAD_REQUEST" to group
+    Examples:
+      | parameterToUpdate     | updateDate        |
+      |                       | change name group |
+      | name                  |                   |
+      | visibility            | null              |
+      | visibility            | Public            |
+      | isArchived            |                   |
+      | isAutoArchiveDisabled |                   |
+      | isBroadcast           |                   |
+      | canHaveChatterGuests  | true              |
```

Comment:
Should be to add one line at the end of the file

Reply:
ok